service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/StandardDirectionsOrderControllerAboutToSubmitTest.java

```diff
@@ -164,25 +182,48 @@ void shouldPopulateStandardDirectionOrderObjectFromUploadRoute() {
     }
 
     @Test
-    void shouldUpdateStateWhenOrderIsSealedThroughServiceRouteAndRemoveRouter() {
+    void shouldUpdateStateWhenOrderIsSealedThroughServiceRouteAndRemoveRouterAndSendNoticeOfProceedings() {
         CaseDetails caseDetails = validSealedCaseDetailsForServiceRoute();
+        CaseData caseData = mapper.convertValue(caseDetails.getData(), CaseData.class);
 
-        AboutToStartOrSubmitCallbackResponse response = postAboutToSubmitEvent(caseDetails);
+        AboutToStartOrSubmitCallbackResponse response = postAboutToSubmitEvent(caseData);
+
+        CaseData responseCaseData = mapper.convertValue(response.getData(), CaseData.class);
+
+        DocumentReference noticeOfProceedingBundle = responseCaseData.getNoticeOfProceedingsBundle().get(0).getValue()
+            .getDocument();
+
+        assertThat(responseCaseData.getNoticeOfProceedingsBundle()).hasSize(1);
+        assertThat(noticeOfProceedingBundle.getUrl()).isEqualTo(DOCUMENT.links.self.href);
+        assertThat(noticeOfProceedingBundle.getFilename()).isEqualTo(DOCUMENT.originalDocumentName);
+        assertThat(noticeOfProceedingBundle.getBinaryUrl()).isEqualTo(DOCUMENT.links.binary.href);
 
         assertThat(response.getData())
             .containsEntry("state", "PREPARE_FOR_HEARING")
             .doesNotContainKey("sdoRouter");
     }
 
     @Test
-    void shouldUpdateStateWhenOrderIsSealedThroughUploadRouteAndRemoveRouter() throws Exception {
+    void shouldUpdateStateWhenOrderIsSealedThroughUploadRouteAndRemoveRouterAndSendNoticeOfProceedings()
+        throws Exception {
         DocumentReference document = DocumentReference.builder().filename("final.pdf").build();
         CaseDetails caseDetails = validCaseDetailsForUploadRoute(document, SEALED);
+        CaseData caseData = mapper.convertValue(caseDetails.getData(), CaseData.class);
 
         given(idamClient.getUserInfo(anyString())).willReturn(UserInfo.builder().name("adam").build());
         given(sealingService.sealDocument(document)).willReturn(document);
 
-        AboutToStartOrSubmitCallbackResponse response = postAboutToSubmitEvent(caseDetails);
+        AboutToStartOrSubmitCallbackResponse response = postAboutToSubmitEvent(caseData);
+
+        CaseData responseCaseData = mapper.convertValue(response.getData(), CaseData.class);
+
+        DocumentReference noticeOfProceedingBundle = responseCaseData.getNoticeOfProceedingsBundle().get(0).getValue()
```

Comment:
They can only upload one C6 and C6a right? Do we want to change this to be a bit more friendly instead of being a list? We could have 2 doc ref fields one for C6 and one for C6a in a complex type and then just add the appropriate document to those fields, then field names will make more sense in the tab. Of course this is under the assumption that only one of each can exist at a time, if not then as we have now is probs the better way to go.

Reply:
Discussed and leaving as is.